app/javascript/modules/listingDetails/ListingDetailsEligibility.tsx

```diff
@@ -380,7 +385,13 @@ export const ListingDetailsEligibility = ({
                   }}
                   buttonClassName="mt-2 has-toggle"
                 >
-                  {stripMostTags(listing.Credit_Rating)}
+                  {stripMostTags(
+                    getTranslatedString(
+                      listing?.Credit_Rating,
```

Comment:
This field is coming from the Salesforce Cache, which is not refreshed at the same time as the translations

Reply:
Will be addressed in https://sfgovdt.jira.com/browse/DAH-2575